File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.c
int finalValueAfterOperations(char** operations, int operationsSize) {
    int a=0;
    for(int i=0;i<operationsSize;i++)
    {
        if(operations[i][1]=='+' )
        {
            a+=1;
        }
        else if(operations[i][1]=='-')
        {
            a-=1;
        }
    }
    return a;
}

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int x=0;
        for(int i=0;i<operations.length;i++)
        {
            String ch=operations[i];
            if(ch.equals("--X") || ch.equals("X--"))
            {
                x-=1;
            }
            else
            {
                x+=1;
            }
        }
        return x;
    }
}